### PR TITLE
added new function create_api

### PR DESCRIPTION
**Pull request by moezali1**
Added a new function `create_api` in `pycaret.classification` and `pycaret.regression`

```
from pycaret.datasets import get_data
data = get_data('boston')

from pycaret.regression import *
s = setup(data, target = 'medv', session_id = 123)

et = create_model('et')

create_api(et, 'boston_api')

!python boston_api.py
```
localhost:8000/docs

image

image

image

image

### CODE REVIEW

File: pycaret/classification.py [hunk 1]
```diff
@@ -2809,3 +2809,44 @@ def convert_model(estimator, language: str = "python") -> str:
...
+
+    port: int, default = 8000
+        port for API.
+
+
+    Returns:
+        None
+
+    """
+    return pycaret.internal.tabular.create_api(estimator=estimator, api_name = api_name, host = host, port = port)
```
Comment (Yard1) [inline pycaret/classification.py:2852]:
```suggestion
    return pycaret.internal.tabular.create_api(estimator=estimator, api_name = api_name, host = host, port = port)

```

File: pycaret/regression.py [hunk 2]
```diff
@@ -2526,3 +2526,44 @@ def convert_model(estimator, language: str = "python") -> str:
...
+
+    port: int, default = 8000
+        port for API.
+
+
+    Returns:
+        None
+
+    """
+    return pycaret.internal.tabular.create_api(estimator=estimator, api_name = api_name, host = host, port = port)
```
Comment (Yard1) [inline pycaret/regression.py:2569]:
```suggestion
    return pycaret.internal.tabular.create_api(estimator=estimator, api_name = api_name, host = host, port = port)

```

**Review by Yard1 — APPROVED** [review]
Can you format it with black? LGTM otherwise

**Comment by ajayarunachalam** [comment]
Problem still persist with the create_api func. Pls. check.

**Comment by MaximeTut** [comment]
There is a problem in the api file when creating an API, the code calling the model is : 
`model = load_model('api')`

instead of : 
`model = load_model('my_best_pipeline')`

**Pull request merged**